src/system/WatchableEventManager.h

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
Derived from `SystemSockets.h`; see https://gist.github.com/kpschoedel/ecc397e3417f98f783eab73c475ebd25 for diff